### PR TITLE
stern: add page

### PR DESCRIPTION
**Pull request by Aracki**
I have added [stern](https://github.com/wercker/stern/) - a tool for tailing Kubernetes logs.

### CODE REVIEW

File: pages/common/stern.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Tail all pods within a current namespace:
+
+`stern .`
+
+- Tail all pods with status waiting:
+
+`stern . --container-state {{waiting}}`
+
+- Tail all pods that matches regular expression:
```
Comment (schneiderl) [inline pages/common/stern.md]:
```suggestion
- Tail all pods that matches a given regular expression:
```

File: pages/common/stern.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`stern {{pod_query}}`
+
+- Tail matched pods from all namespaces:
+
+`stern {{pod_query}} --all-namespaces`
+
+- Tail matched pods from 15m ago:
```
Comment (schneiderl) [inline pages/common/stern.md]:
```suggestion
- Tail matched pods from 15 minutes ago:
```

File: pages/common/stern.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+> Tail multiple pods and containers from Kubernetes.
+> More information: <https://github.com/wercker/stern/>.
+
+- Tail all pods within a current namespace:
+
+`stern .`
+
+- Tail all pods with status waiting:
+
+`stern . --container-state {{waiting}}`
```
Comment (agnivade) [inline pages/common/stern.md]: Is there a fixed set of statuses ? Then we can do `waiting|started|stopped` for eg. And change the description to "Tail all pods with a given status"

Comment (Aracki) [reply pages/common/stern.md]: Oh, I forgot the syntax for that. Check if it's ok now?

**Comment by Aracki** [comment]
@mebeim Thank you 😄 

They all must be stored in tldr 😈

**Pull request merged**